.github/workflows/continuous_integration.yml

```
@@ -63,6 +67,8 @@ jobs:
         uses: taiki-e/install-action@cargo-llvm-cov
       - name: Generate code coverage
         run: cargo llvm-cov --all-features --workspace --lcov --output-path lcov.info
+        env:
+          - STRIPE_KEY: any-key-test
```

actionlint (1.7.10)

"env" section is sequence node but mapping node is expected

(syntax-check)